lib/customer-deposit-wallet/http/Cardano/Wallet/Deposit/HTTP/Types/API.hs

```diff
@@ -37,3 +39,7 @@ type CustomerAPI =
         "customers"
             :> Capture "customerId" (ApiT Customer)
             :> Verb 'PUT 200 '[JSON] (ApiT Address)
+
+type NetworkAPI =
+        "network" :> "tip"
```

Comment:
Also, we have to be careful about naming here — `getWalletTip` does **not** return the tip of the network, but the point until which the wallet has synchronized. 🤔

Reply:
indeed, proposed `/network/synchronization-point` which seems to be very clear and correct IMHO

Reply:
Uh, our typical nomenclature for this is `localTip` of the wallet. 😅 I think that just `/localTip`, `/walletTip` or even `/tip` would be appropriate.